samples/drivers/led_gpio/src/main.c

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2018 Savoir-Faire Linux.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <device.h>
+#include <errno.h>
+#include <drivers/led.h>
+#include <sys/util.h>
+#include <sys/printk.h>
+#include <zephyr.h>
+
+#define LED_DEV_NAME DT_GPIO_LEDS_LEDS_DRIVER
+#define NUM_LEDS 4
```

Comment:
Why 4? Not all boards will have 4 LEDs. I'll suggest using 2.

Reply:
Right. I reduce LEDs count to 2.